test/storybook/visual-regression/v-checkbox.spec.ts

```diff
@@ -1,27 +1,93 @@
 import { test, expect } from "@playwright/test"
 
+test.describe.configure({ mode: "parallel" })
+
```

Comment:
It's so nice that the snapshots are smaller now!

Reply:
Yeah, it's hard to see the diffs when the actual content is a small part of the snapshot.